sympy/polys/rootconditions.py

```diff
@@ -127,11 +121,23 @@ def _clear_cond_poly(cond: PolyElement[Er], previous_cond, K):
 
     cond = prod([fac for fac, m in facs_m if m % 2 == 1], K.one)
     if common_factor < 0:
-        cond = -K.one* cond
+        cond = -K.one*cond
 
     return cond
 
 
+def is_totally_ground(c: PolyElement[Er]) -> bool:
+    """Check if the condition is ground in all subrings"""
+    while True:
+        if not c.is_ground:
+            return False
+        ring = c.ring
+        domain = ring.domain
+        if not domain.is_PolynomialRing:
+            return True
+        c = c.drop(*ring.gens)
```

Comment:
I'm not sure this is really necessary but it would be:
```python
def is_totally_ground(c: PolyElement[Er]) -> bool:
    while c.ring.domain.is_PolynomialRing and c.is_ground:
        c = c.LC
    return c.is_ground
```
The drop method is inefficient and should not be used.